server/views/pages/detail.njk

```diff
@@ -10,7 +10,7 @@
 {% block content %}
   <div class="govuk-grid-row govuk-body">
     <div class="govuk-grid-column-three-quarters">
-      <h1>{{ personalDetails.firstName | capitalize }} {{ personalDetails.middleName | capitalize }} {{ personalDetails.lastName }}</h1>
+      <h1>{{ personalDetails.firstName | title }} {{ personalDetails.middleName | title }} {{ personalDetails.lastName }}</h1>
```

Comment:
turns out I got confused - `title` ensures that `JOHN JAMES` ends up as `John James` rather than `John james`